services/storage/src/simcore_service_storage/simcore_s3_dsm.py

```diff
@@ -783,43 +768,14 @@ async def create_soft_link(
         target.file_id = link_file_id  # NOTE: api-server relies on this id
         target.is_soft_link = True
 
-        async with self.engine.acquire() as conn:
-            return convert_db_to_model(await db_file_meta_data.insert(conn, target))
-
-    async def synchronise_meta_data_table(
```

Comment:
Q: so this is no longer required?
I don't remember why this is the case.

Reply:
nobody uses it.